app/views/providers/legal_aid_applications/_legal_aid_applications.html.erb

```diff
@@ -1,30 +1,52 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <table class="govuk-table">
+    <table class="govuk-table sortable table-merge_columns">
       <caption class="govuk-table__caption"><%= t('.current_application') %></caption>
       <thead class="govuk-table__head">
         <tr class="govuk-table__row">
-          <th class="govuk-table__header" scope="col"><%= t('.applicant_name') %></th>
-          <th class="govuk-table__header" scope="col"><%= t('.created_at') %></th>
-          <th class="govuk-table__header" scope="col"><%= t('.application_ref') %></th>
-          <th class="govuk-table__header" scope="col"><%= t('.status') %></th>
-          <th class="govuk-table__header" scope="col"></th>
+          <%= sort_column_th type: :alphabetic, content: t('.applicant_name'),   currently_sorted: @initial_sort[:applicant_name] %>
+          <%= sort_column_th type: :date,       content: t('.created_at'),       currently_sorted: @initial_sort[:created_at] %>
+          <%= sort_column_th type: :alphabetic, content: t('.application_ref'),  currently_sorted: @initial_sort[:applicant_ref] %>
+          <%= sort_column_th type: :alphabetic, content: t('.certificate_type'), currently_sorted: @initial_sort[:certificate_type] %>
+          <%= sort_column_th type: :alphabetic, content: t('.status'),           currently_sorted: @initial_sort[:status] %>
```

Comment:
Can we have a _combine right_ value here, with associated stuff?
Probably combine `application_ref` and `certificate_type`.
At present the table goes off the right when narrow.